src/engine/LoadingScreen.cs

```diff
@@ -31,11 +33,17 @@ public class LoadingScreen : Control
     private Label tipLabel;
     private Control spinner;
 
-    private string loadingMessage = TranslationServer.Translate("LOADING");
+    private bool visible;
+
+    private Random random;
+    private Timer randomizeTipTimer;
```

Comment:
I think that Godot timer node should be used instead of system timer as the godot timer is more integrated with pausing etc.